osu.Game/Screens/Play/HUD/HitErrorMeters/BottomBarHitErrorMeter.cs

```diff
@@ -0,0 +1,286 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.Scoring;
+using osuTK;
+using osuTK.Graphics;
+using System.Linq;
+
+namespace osu.Game.Screens.Play.HUD.HitErrorMeters
+{
+    public class BottomBarHitErrorMeter : HitErrorMeter
```

Comment:
Why is this a complete copy-paste? It should 100% be able to share at least 99% of logic with the side bsrs.

Reply:
Decided to go with the copy-paste way for the sake of code readability (else code would be filled with inline conditionals) . Will refactor to at least share used components.